google-cloud-clients/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/FormatOptions.java

```diff
@@ -26,15 +26,25 @@
 /**
  * Base class for Google BigQuery format options. These class define the format of external data
  * used by BigQuery, for either federated tables or load jobs.
+ *
+ * Load jobs support the following formats:
```

Comment:
Thanks. Since separate classes would be a breaking change (that's probably not worth it), I think documentation is the best way.